flink-table-store-core/src/test/java/org/apache/flink/table/store/file/mergetree/MergeTreeTest.java

```diff
@@ -61,7 +61,12 @@
 import static java.util.Collections.singletonList;
 import static org.assertj.core.api.Assertions.assertThat;
 
-/** Test for {@link MergeTree}. */
+/**
+ * Test for {@link MergeTree}.
+ *
+ * <p>Manual test: please adjust TARGET_FILE_SIZE to 1, so that a large number of upgrade files will
+ * be generated.
```

Comment:
No need for manual test. For `testCloseUpgrade` set write buffer size to 4kb, page size to 1kb and target file size to 1kb. With 1000 records and `@RepeatedTest(10)` to run 10 times it has quite some probability to fail without this fix.

Reply:
I will use ParameterizedTest